esp-wifi/src/ble/os_adapter_esp32c3.rs

```diff
@@ -81,7 +81,7 @@ pub(super) static G_OSI_FUNCS: osi_funcs_s = osi_funcs_s {
     magic: 0xfadebead,
     version: 0x00010006,
     interrupt_set: Some(ble_os_adapter_chip_specific::interrupt_set),
-    interrupt_clear: Some(ble_os_adapter_chip_specific::interrupt_clear),
+    clear_interrupt: Some(ble_os_adapter_chip_specific::clear_interrupt),
```

Comment:
these should probably stay consistent with the C side :(

Reply:
Oops, good spot! Thanks!